### PR TITLE
Feat/align vcs single file content

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
This PR aligns vcs/single-file-content capability with our conventions
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: capabilities/vcs/single-file-content/maps/gitlab.suma [hunk 1]
```diff
@@ -10,13 +10,16 @@ map SingleFileContent{
   ref = input.ref
 
   http GET "/projects/{id}/repository/files/{path}?ref={ref}" {
-    response 200 "application/json" {
-      //map returned values to result object
-      map result {
+     response  {
```
Comment (freaz) [inline capabilities/vcs/single-file-content/maps/gitlab.suma:13]: Nitpick 

```suggestion
     response {
```

File: capabilities/vcs/single-file-content/maps/gitlab.test.ts [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
-    const value = result.unwrap();
+  describe('SingleFileContent', () => {
+    it('should perform successfully', async () => {
+      await expect(
+        superface.run({
+          profile: 'vcs/single-file-content',
+          provider: 'gitlab',
+          useCase: 'SingleFileContent',
+          input: {
+            owner: 'Jakub-Vacek',
```
Comment (freaz) [inline capabilities/vcs/single-file-content/maps/gitlab.test.ts:18]: For github you use astexplorer, does make sense to hire unified?

File: capabilities/vcs/single-file-content/profile.supr [hunk 3]
```diff
@@ -1,37 +1,75 @@
...
     content! string!
 
-    "Used encoding"
+    """
+    Encoding
+    Used encoding
+    """
     encoding! string!
   }
+  error {
```
Comment (freaz) [inline capabilities/vcs/single-file-content/profile.supr:58]: Adding error to profile should result in minor version update as error handling changes in consumers applications.

**Review by freaz — CHANGES_REQUESTED** [review]
Few minor things, but profile version should be raised in minor except patch.

**Review by zdne — APPROVED** [review]
🙌🏻

**Pull request merged**